Module-generation/frontmatter.py

```diff
@@ -14,89 +11,63 @@
         getModuledata = open(moduleFile,"r")
         readModuleData=getModuledata.read()
         parseData=json.loads(readModuleData)
-        
-    
         moduleName=parseData['id']
         moduleDescription=parseData['description']
         
-       
-        
+        moduleLogo="logo.jpg"
+        moduleReadme=DIR+folders+"/README.md"
 
+        os.mkdir(DST+moduleName)
+        IndexMd=open(DST+moduleName+"/index.md","a+")
+        IndexMd.write('---\n')
+        IndexMd.write('posttype: "module" \n')
+        IndexMd.write('title: '+moduleName+'\n')
+        IndexMd.write('description: "'+moduleDescription+'"\n')
+
+        if(os.path.isfile(moduleLogo)):
```

Comment:
what if there's no logo available? did I overlook the `else` path?

Reply:
This is just a small check, handled on scrape.py

Reply:
then at least provide a comment here, that explains this.